HeartbeatLogging/Tests/Unit/HeartbeatStorageTests.swift

```diff
@@ -285,8 +285,10 @@ private class StorageFake: Storage {
   func read() throws -> Data {
     if let onRead = onRead {
       return try onRead()
+    } else if let data = data {
+      return data
     } else {
-      return try data ?? { throw StorageError.readError }()
```

Comment:
This was reworked to avoid an error where running `swiftformat` on the file broke the build.

I filed https://github.com/nicklockwood/SwiftFormat/issues/1142